tests/unit/addons/message2.json

```diff
@@ -4,13 +4,14 @@
   "name": "Test Message 2",
   "translatable": false,
   "type": "message",
-  "conditions": { },
+  "conditions": {
+    "end_time": 1599424544
```

Comment:
this is in the past. So message2 will not be loaded.

Reply:
yeap that is on purpose. that was the way i found to load the message and be sure it is disabled.